internal/pkg/template/templates/addons/aurora/rdws/serverlessv2.yml

```diff
@@ -135,6 +135,7 @@ Resources:
       {{- end}}
       DBClusterParameterGroupName: {{- if .ParameterGroup}} {{.ParameterGroup}} {{- else}} !Ref {{logicalIDSafe .ClusterName}}DBClusterParameterGroup {{- end}}
       DBSubnetGroupName: !Ref {{logicalIDSafe .ClusterName}}DBSubnetGroup
+      Port: {{if eq .Engine "MySQL"}}3306{{else}}5432{{end}}
```

Comment:
wut but from the commit description it looks like RDS does this by default? what am I misunderstanding?

Reply:
Commit msg is a lie...

Reply:
Basically we have two ways of fixing the problem: change the db cluster port or change the sg port. The first one makes more sense because 5432 is the default port for postgre, instead of making it always to be 3306.

Reply:
> Instead of making it always to be 3306.

I see what you're saying that right now regardless of the engine being postgres or mysql, it defaults to 3306.

Whereas it should have defaulted to 5432 if it was postgres. Since it doesn't do that, you have to explicitly assign it.

Am I understanding correctly?

Reply:
Yeah precisely. It should work regardless of how they change the default port on the service side.